### ISSUE TITLE
Does work Batch normalization when Batchnormalization=false to true

### ISSUE THREAD

**Issue by agpoyraz** [vlfeat/matconvnet#996]
In the MNIST example batchnormalization=true. But VL_NNBNORM() function has nothing(just comment lines). Is it working though this? or Am i have to change vl_nnbnorm function?
Thanks..

**Comment by ankitkp4056** [comment]
I guess it is working, because even the vl__nnconv() function just has comment lines.

**Comment by agpoyraz** [comment]
You are right thank you. Probably it works. But the second question is how is it working
